bundles/persistence/org.openhab.persistence.sitewhere/src/main/java/org/openhab/persistence/sitewhere/internal/SiteWhereConfigurationManager.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.persistence.sitewhere.internal;
+
+import java.io.File;
+
+import org.openhab.config.core.ConfigDispatcher;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Manages SiteWhere configuration options.
+ * 
+ * @author Derek
+ */
+public class SiteWhereConfigurationManager {
```

Comment:
the default configuration mechanism in OH1 is the openhab.cfg file. Any reason why you've choosen a special (and proprietary) installation?